SilKit/source/core/participant/Participant_impl.hpp

```diff
@@ -1328,7 +1328,7 @@ template <typename SilKitConnectionT>
 template <typename MessageT>
 void Participant<SilKitConnectionT>::HandleSynchronizationPoint()
 {
-    if constexpr (SilKitMsgTraits<typename RemoveCvRef<MessageT>::type>::IsSynchronizationPoint())
+    if constexpr (SilKitMsgTraits<typename RemoveCvRef<MessageT>>::IsSynchronizationPoint())
```

Comment:
You should also not need the `typename`, since the compiler knows that `RemoveCvRef` is a type.

```suggestion
    if constexpr (SilKitMsgTraits<RemoveCvRef<MessageT>>::IsSynchronizationPoint())
```